float64 wheelRadius
float64 wheelDistance
